packages/pass-style/src/error.js

```diff
@@ -8,7 +8,103 @@ import { Fail, q, hideAndHardenFunction } from '@endo/errors';
  * @import {PassStyle} from './types.js';
  */
 
-const { getPrototypeOf, getOwnPropertyDescriptors, hasOwn, entries } = Object;
+const {
+  defineProperty,
+  getPrototypeOf,
+  getOwnPropertyDescriptors,
+  getOwnPropertyDescriptor,
+  hasOwn,
+  entries,
+  freeze,
+} = Object;
+
+const { apply } = Reflect;
+
+const hardenIsFake = () => {
+  // We do not trust isFrozen because lockdown with unsafe hardenTaming replaces
+  // isFrozen with a version that is in cahoots with fake harden.
+  const subject = harden({ __proto__: null, x: 0 });
+  const desc = getOwnPropertyDescriptor(subject, 'x');
+  return desc?.writable === true;
+};
+
+const makeRepairError = () => {
+  if (!hardenIsFake()) {
+    return undefined;
+  }
+
+  const er1StackDesc = getOwnPropertyDescriptor(Error('er1'), 'stack');
```

Comment:
I'm curious why we didn't build an error by syntax instead for one of the candidates, to be resilient against global `Error` override.

Reply:
I do not see such a commit